pkg/flow/reconciler/reconcile_adapter.go

```diff
@@ -52,16 +53,25 @@ func MakeObsEnv(cfg source.ConfigAccessor) []corev1.EnvVar {
 }
 
 // MakeAdapterLabels adds generic label generation
-func MakeAdapterLabels(adapterName, name string) labels.Set {
+func MakeAdapterLabels(adapterName string, object kmeta.OwnerRefable) labels.Set {
```

Comment:
I think it would be cleaner if we kept a clear separation of concerns:
- generate a list of generic labels
- propagate labels from an external source

Those two things shouldn't be mixed an matched in a single function IMO.

(comment applies to all changed components)

Reply:
Well, then the function's name `MakeAdapterLabels` is quite confusing. Let me see how it can be changed.

Reply:
You right, I was mixing that up with the following code in the sources and routers:

https://github.com/triggermesh/triggermesh/blob/5499508d0f8f351d8b2f825f57b4e26e7e6113f2/pkg/sources/reconciler/common/adapter.go#L247-L256

Considering that we don't have a common way to reconcile things in TriggerMesh, I believe it's pointless to discuss style in this PR.
Feel free to ignore.